Andraquin/iReporter
# #162411545 Implement feedback

#### What does this PR do?
Implement feedback
#### Description of Task to be completed?
- move all red-flags controllers into one file
- delete the red-flags controller folder
- configure body-parser in server/index.js file
- add a command to .eslintrc file
- add link to Heroku API endpoints
- add link to UI template
- add link to pivotal tracker board
- setup section
- add instructions on how to run test
- add technologies and tools used
- update test suite to check for actual response values
- update middleware to return all errors at once
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[162411545](https://www.pivotaltracker.com/story/show/162411545)
